api/v4/source/scheduled_post.yaml

```diff
@@ -58,3 +58,34 @@
           $ref: "#/components/responses/Forbidden"
         "500":
           $ref: "#/components/responses/InternalServerError"
+
+  /api/v4/posts/scheduled/team/{team_id}:
+    get:
+      tags:
+        - scheduled_post
+      summary: Gets all scheduled posts for a user for the specified team, include those belonging to DMs and GMs.
+      description: >
+        Get user-team scheduled posts
+
+        ##### Permissions
+
+        Must have `access_team` permission for the team the scheduled posts are being fetched for.
+
+      operationId: GetUserScheduledPosts
+      responses:
+        "200":
+          description: Created scheduled post
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/ScheduledPost"
```

Comment:
From the last changes, this should be updated too, right?

Reply:
Done. I missed this.